data/Dockerfiles/sogo/Dockerfile

```diff
@@ -8,6 +8,7 @@ ENV GOSU_VERSION 1.9
 # Prerequisites
 RUN apt-get update && apt-get install -y --no-install-recommends \
 		apt-transport-https \
+		bind9-host \
```

Comment:
Actually, I chose to use bind9 to match the acme container, which also uses `dig` (that's the only other place in Mailcow that performs an explicit DNS lookup).
From a technical perspective, it makes no difference whether we use `dig` or `unbound-host`, so we can just keep `dig` here.